modules/services/parcellite.nix

```diff
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.parcellite;
+
+in
+
+{
+  meta.maintainers = [ maintainers.gleber ];
+
+  options = {
+    services.parcellite = {
+      enable = mkEnableOption "Parcellite";
+
+      package = mkOption {
```

Comment:
I don't see multiple Parcellite packages in Nixpkgs. Is this option necessary?

Reply:
Dropped.